Bump to v9.0.1 (#546)
PostgREST/postgrest-docs: docs/releases/v9.0.1.rst

PostgREST 9.0.1
===============

> This version includes some bug fixes. You can download the pre-compiled binaries on the `GitHub release page <https://github.com/PostgREST/postgrest/releases/tag/v9.0.1>`_.

Reviewer: ```suggestion
This version includes important fixes for production environments and other miscellaneous bug fixes. You can download the pre-compiled binaries on the `GitHub release page <https://github.com/PostgREST/postgrest/releases/tag/v9.0.1>`_.
```

> This version includes important fixes for production environments and other miscellaneous bug fixes. You can download the pre-compiled binaries on the `GitHub release page <https://github.com/PostgREST/postgrest/releases/tag/v9.0.1>`_.

Reviewer: ```suggestion
This version includes important fixes for production environments and other miscellaneous fixes. You can download the pre-compiled binaries on the `GitHub release page <https://github.com/PostgREST/postgrest/releases/tag/v9.0.1>`_.
```

Bug Fixes
---------

* Keep working when ``EMFILE (Too many open files)`` is reached. (`#2042 <https://github.com/PostgREST/postgrest/issues/2042>`_)

> * Disable parallel GC for better performance on higher core CPUs. (`#2294 <https://github.com/PostgREST/postgrest/issues/2294>`_)

Reviewer: ```suggestion
* Disable parallel GC for better performance on higher core CPUs (`#2294 <https://github.com/PostgREST/postgrest/issues/2294>`_). Thanks to `NoRedInk for their blog post <https://blog.noredink.com/post/666654908557180928/tuning-haskell-rts-for-kubernetes-part-2>` that lead us to this fix.
```
